pkg/intermediate/aggregate.go

```diff
@@ -984,3 +995,29 @@ func isCorrelationRequired(flowType uint8, record entities.Record) bool {
 	}
 	return false
 }
+
+func fillHttpVals(incomingHttpVals, existingHttpVals string) (string, error) {
```

Comment:
Can we add a unit test for this function to make sure we correlate two HttpVals correctly?